dev/README_RELEASE_PYTHON_CLIENT.md

```diff
@@ -135,11 +120,30 @@ breeze release-management prepare-python-client --package-format both --python-c
 ```shell script
 cd ${CLIENT_REPO_ROOT}
 git diff HEAD
+git checkout -b release-${VERSION}
 git add .
 git commit -m "Update Python Client to ${VERSION}${VERSION_SUFFIX}"
-git push origin main
+git push apache release-${VERSION}
 ```
 
+Then open a PR and merge it into main.
+
+- Tag your release with RC candidate tag (note that this is the RC tag even if version of the packages
```

Comment:
We were tagging too early in the process, before we'd even generated the client.